docs/getting-started/launching-one-node/crypto-materials.md

```diff
@@ -9,3 +9,66 @@ title: Creating Crypto Material
  SPDX-License-Identifier: CC-BY-4.0
  -->
 
+### Crypto materials configuration
+
+Hyperledger Orion uses public key cryptography from user side, for user authentication and transaction Non-Repudiation. From server side, Orion uses PKI to sign and verify all server responses to clients. 
```

Comment:
Hyperledger Orion uses public key cryptography to achieve authentication and transaction non-repudiation. This holds for both the client and the server. Therefore, each client signs the queries and transactions it submits to the server, and each server signs the responses it sends to the client.